components/youtube/new-videos.js

```diff
@@ -0,0 +1,65 @@
+const axios = require('axios')
+const youtube = {
+  type: "app",
+  app: "youtube_data_api",
+}
+
+
+module.exports = {
+  name: "Youtube - New Videos",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    youtube,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+  methods: {
+    async getVideos(pageToken=null) {
+      return await axios.get('https://www.googleapis.com/youtube/v3/search', {
+        headers: {
+          'Authorization' : 'Bearer '+this.youtube.$auth.oauth_access_token
```

Comment:
Prefer template literals over concatenation:

```
`Bearer ${this.youtube.$auth.oauth_access_token}`
```

(when you move this into the app file, the reference to the access token will also change to just `this.$auth.oauth_access_token`)

Reply:
Updated this.